### ISSUE TITLE
Double invoice for order

### ISSUE THREAD

**Issue by gijsde3de** [MultiSafepay/Magento2Msp#67]
We're running on msp 1.4.7 and some orders get 2 paid invoices that are identical.
In the logs we see the following calls 

```
{"time":"2018-04-18T18:47:00+00:00", "remote_addr":"185.99.128.21", "remote_user":"-", "host":"www.xxx.nl", "request":"GET /multisafepay/connect/notification/?transactionid=000000962&timestamp=1524077219 HTTP/1.1", "status":"200", "body_bytes_sent":"12", "referer":"-", "user_agent":"Mozilla/5.0 (X11; Debian; Linux x86_64; rv:33.0) Gecko/20100101", "request_time":"0.548", "handler":"phpfpm", "country":"NL", "port":"443", "ssl_cipher":"xxx", "ssl_protocol":"TLSv1.2"}

{"time":"2018-04-18T18:47:35+00:00", "remote_addr":"185.99.128.21", "remote_user":"-", "host":"www.xxx.nl", "request":"GET /multisafepay/connect/notification/?transactionid=000000962&timestamp=1524077253 HTTP/1.1", "status":"200", "body_bytes_sent":"12", "referer":"-", "user_agent":"Mozilla/5.0 (X11; Debian; Linux x86_64; rv:33.0) Gecko/20100101", "request_time":"1.145", "handler":"phpfpm", "country":"NL", "port":"443", "ssl_cipher":"xxx", "ssl_protocol":"TLSv1.2"}
```

Both the invoices created_at time is 18:47:34 any suggestions on how to fix this issue?

**Comment by Tjitse-E** [comment]
Related issue #72

**Comment by Esteraa** [comment]
We have the same problem after updating to M2.2.6. Currently we are running on MSP 1.6.0. 

@jasper-multisafepay Are there any solutions to this problem?

**Comment by Jasper-MultiSafepay** [comment]
We would really need access to a staging environment, where we can reproduce this reported issue, as we cannot reproduce on our test platforms.
Could you email us at techsupport@multisafepay.com with the required credentials, so we can investigate this further?

**Comment by Esteraa** [comment]
@jasper-multisafepay We fixed this with #77. No need to investigate our email anymore.
